src/SWP/Bundle/ContentBundle/Controller/ArticleController.php

```diff
@@ -55,10 +60,19 @@ class ArticleController extends Controller
      */
     public function listAction(Request $request)
     {
+        $authors = '';
+        if (null !== $request->query->get('author', null)) {
+            $authors = explode(', ', $request->query->get('author'));
+        }
+
         $articles = $this->get('swp.repository.article')
             ->getPaginatedByCriteria(new Criteria([
-                'status' => $request->query->get('status', ''),
+                'status' => $request->query->get('status', ArticleInterface::STATUS_PUBLISHED),
```

Comment:
Hmm... from other side - it's internal API. I woud expect to see only bublished allways, but then we would need add option to get all of them. So ok, i will revert this one.